Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG SNAP_RISK=stable
+ARG SNAP_TRACK=latest
+ARG SNAP_ARCH
+ARG SNAP_VERSION
+ARG BASE_ARCH
+
+FROM ubuntu:18.04
```

Comment:
let's make this 20.04
(we use core20 for the 4.4 snap)

Reply:
I've bumped the version, but this is the build image. Since this pr is for 4.0 it will remain 18.04 for the target images.